guide/08 Finding places with Geocoding/geocoding module.ipynb

```diff
@@ -0,0 +1,80 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# geocoding module\n",
+    "\n",
+    "#### A Brief Introduction to Geocoding\n",
+    "\n",
+    "In a GIS, geocoding is the process of transforming various place-name descriptions into a location on the earth's surface. For successful geocoding operations there are necesary component technologies that must meet certain requirements before an place description can be properly located. The [ArcGIS Desktop Geocoding Documentation](http://desktop.arcgis.com/en/arcmap/latest/manage-data/geocoding/what-is-geocoding.htm) explains the necessary components to successfully complete geocoding operations. It also outlines [Essential geocoding vocabulary](http://desktop.arcgis.com/en/arcmap/latest/manage-data/geocoding/essential-geocoding-vocabulary.htm) to help you succeed in using the geocoding classes and functions in the ArcGIS API for Python geocoding module.\n",
+    "\n",
+    "#### Geocoding Module Types and Functions\n",
+    "\n",
+    "The geocoding module provides a `Geocoder` class to construct `geocoder` objects from portal geocoding service items or from the url to a [geocoding service]((https://server.arcgis.com/en/server/latest/publish-services/linux/geocode-services.htm). In addition, the `get_geocoders` function returns a list of `geocoders` accessible in a portal. \n",
```

Comment:
I would prefer using the generic term 'GIS' instead of portal. Portal is only available with ArcGIS Enterprise and not with ArcGIS Online.